h5g_group_test.go

```diff
@@ -17,8 +17,7 @@ func TestGroup(t *testing.T) {
 	defer os.Remove(fname)
 	defer f.Close()
 
-	err = f.CheckLink("/foo")
-	if err == nil {
+	if f.LinkExists("/foo") {
 		t.Fatalf("err: %s", "/foo shouldn't exist at this time")
```

Comment:
There is no error here.

Reply:
This one should be an error because if the /foo already existed before it was created than that is not quite right

Reply:
The condition being tested is not an error condition.
```
t.Fatal("unexpected /foo link present")
```